### ISSUE TITLE
Incorrect switch option in help

### ISSUE THREAD

**Issue by uberhacker** [saltedcoffii/compressdoc#3]
While testing this, I noticed `--bzip2, --bz2, -j` in the man page.  Change the `-j` to `-b` instead?

**Comment by saltedcoffii** [comment]
I was puling -j from `tar --help`'s output, see here:
```
  -j, --bzip2                filter the archive through bzip2
  -J, --xz                   filter the archive through xz
      --lzip                 filter the archive through lzip
      --lzma                 filter the archive through xz
      --lzop                 filter the archive through lzop
      --no-auto-compress     do not use archive suffix to determine the
                             compression program
      --zstd                 filter the archive through zstd
  -z, --gzip, --gunzip, --ungzip   filter the archive through gzip
  -Z, --compress, --uncompress   filter the archive through compress
```

So I'll change `-b` to `-j` in the compressdoc options

**Comment by saltedcoffii** [comment]
Fixed! see 58eacac
